composer.json

```diff
@@ -3,6 +3,7 @@
     "description": "A PHP class that acts as wrapper for the Converge API",
     "keywords": ["converge", "api", "virtual merchant", "elavon"],
     "homepage": "https://github.com/markroland/converge-api-php",
+    "version": "0.3",
```

Comment:
I'd like to remove this. I prefer to have versions represented as tags. That's Composer's recommendation: https://getcomposer.org/doc/04-schema.md#version